templates/field--field-funding-contact-phone.html.twig

```diff
@@ -0,0 +1,6 @@
+<div class="funding-ct--summary-section--contact-info-phone">
+    {% for item in items %}
+        <span class="wdn-icon-phone" aria-hidden="true"></span><span class="wdn-text-hidden">phone icon</span>{{ item }}
```

Comment:
This field only permits one value at this time. Changing expression to use index 0 instead.